cardinal/config.go

```diff
@@ -42,6 +42,8 @@ type WorldConfig struct {
 	CardinalLogLevel          string  `config:"CARDINAL_LOG_LEVEL"`
 	StatsdAddress             string  `config:"STATSD_ADDRESS"`
 	TraceAddress              string  `config:"TRACE_ADDRESS"`
+	// RouterKey is a JWT token used to secure communications between the game shard and the base shard.
```

Comment:
This is technically not a JWT token, a JWT token would need to be signed by the server during an authentication process. This is more like a normal API key, which is fine, but the comment needs to be corrected.

Reply:
removed `JWT` from the comment